src/app/pages/questions/components/question/audio-input/audio-input.component.ts

```diff
@@ -86,7 +80,17 @@ export class AudioInputComponent implements OnInit, OnChanges {
   startRecording() {
     this.permissionUtil.getRecordAudio_Permission().then(success => {
       if (success === true) {
-        this.audioRecordService.startAudioRecording(this.configFile)
+        this.audioRecordService.startAudioRecording(
+          this.fileName,
+          this.configFile
+        )
+        setTimeout(() => {
+          console.log('Time up for recording')
+          this.audioRecordService.stopAudioRecording()
+          this.audioRecordService
+            .readAudioFile(this.fileName)
+            .then(data => this.valueChange.emit(data))
+        }, this.recordingTime)
```

Comment:
Why is there a hard coded timeout ? What if we add a really long recite text in the definitions, then this will fail.

I think its best to remove this and if it is needed then this timeout should be calculated based on the length of the recite text and not be a constant value.